realm/realm-library/src/main/java/io/realm/Realm.java

```diff
@@ -784,6 +785,82 @@ private Scanner getFullStringScanner(InputStream in) {
     }
 
     /**
+     * Bulk insert of a list of RealmObjects, this is faster than {@link #copyToRealm(Iterable)} since it
```

Comment:
"faster" -> "generally faster" (if a class only has one field, I don't think it will be faster)